_includes/head.html

```diff
@@ -1,11 +1,11 @@
 <head>
   {% capture title %}{% t page.title %}{% endcapture %}
   {% unless title.size > 0 %}{% assign title = page.title %}{% endunless %}
-  {% capture title %}{% if title.size > 0 %}{{ title }} – {% endif %}{{ site.title }}{% endcapture %}
+  {% capture title %}{% if title.size > 0 %}{{ title }} – {{ site.title }}{% else %}{{ site.title }} – {{ t homepage.header }}{% endif %}{% endcapture %}
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>{{ title }}</title>
-  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
+  <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ t homepage.description }}{% endif %}">
```

Comment:
Doesn't that break the description of blog posts because it fixes it to a single, static string?